### ISSUE TITLE
False positive for the TupleParentheses rule when combined with the cons operator

### ISSUE THREAD

**Issue by Volcomix** [fsprojects/FSharpLint#299]
### Description

False positive for the `TupleParentheses` rule when combined with the `::` (cons) operator.

### Repro steps

```fsharp
let foo aStringList = ("bar", "" :: aStringList)
```

### Expected behavior

Should not generate a warning.

### Actual behavior

Generate a warning surrounding only the 2nd tuple element: `Lint: Use parentheses for tuple instantiation.`

### Known workarounds

Disable the `TupleParentheses` rule by following [these instructions](https://github.com/ionide/ionide-vscode-fsharp/issues/997#issuecomment-458079177).
